Assets/tiny_rogue/Scripts/Systems/GameStateSystem.cs

```diff
@@ -289,7 +289,7 @@ public void MoveToInGame( EntityCommandBuffer cb )
 
             GenerateLevel();
             tms.ResetTurnCount();
-            log.AddLog("You enter the dungeon. (Use the arrow keys to explore!)");
+            log.AddLog("You enter the dungeon. (explore:arrows Z:interact X:wait)");
```

Comment:
Let's do "Use the arrow key to explore, z to interact and x to wait" it will fit.

Reply:
It doesn't, I tried. `[space]` is rendered over the top.

Reply:
Ah I meant replace the entire line with that.

... which I then tried and didn't like because it seemed to abrupt so I added a "Welcome!" on the front :)

Reply:
Haha, yeah, fair enough